CONTRIBUTING.md

```diff
@@ -53,9 +53,6 @@ Tips to make your edit/test/debug worflow as productive as possible:
   - pass [CI](http://drone.geirsson.com:8001/scalameta/scalameta), which
     automatically runs when your pull request is submitted.
   - be reformatted with `./bin/scalafmt`
-  - add a relevant entry to the
```

Comment:
Despite the changelog not being updated I think we can keep it. To keep on hoping someone will have time to do it at some point :sweat_smile:

Reply:
Changelog [moved](https://github.com/scalameta/scalameta/blob/master/changelog/2.0.0.md) to GitHub releases. Why store it separately?

Reply:
Ach, alright! Makes sense.